bread/urls.py

```diff
@@ -86,6 +88,16 @@ def get_success_message(self, data):
     path("auth/", include("django.contrib.auth.urls")),
     path("accounts/login/", auth.BreadLoginView.as_view(), name="login"),
     path("accounts/logout/", auth.BreadLogoutView.as_view(), name="logout"),
+    path("accounts/user/", userprofile.UserProfileView.as_view(), name="userprofile"),
+    autopath(
+        userprofile.EditPersonalDataView.as_view(), urlname="userprofile.personal"
```

Comment:
I was just wondering: Would you find it ugly to have this urlname as a static method/variable in the EditPersonalDataView? This would help prevent typos...

Reply:
In short: I think it is simply not necessary.

In long: In case a user needs to use the view he either needs to lookup this hardcoded string identifier or the name of the static variable. So it requires scaning the code anyway to get the correct identifier (be it string or variable) and the typo can happen with a variable name all the same.

Also, we have a good tool for finding names of URLS: ```manage.py show_urls```. This is what I use most often do discover URLs of other apps.

If I want to avoid hardcoded strings for URL names I use autopath without the ```urlname``` parameter.